webdriver/actions.py

```diff
@@ -47,8 +47,8 @@ def key_reporter(session, test_keys_page, request):
 
 
 @pytest.fixture
-def test_keys_page(session, server):
-    session.url = server.where_is("test_keys_wdspec.html")
+def test_keys_page(session, url):
+    session.url = url("/webdriver/support/test_keys_wdspec.html")
```

Comment:
This html file got moved in https://github.com/w3c/web-platform-tests/pull/5024, so you may as well leave it as it was and save yourself from a merge conflict.